src/screens/photo-screen.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {View, TouchableOpacity, StyleSheet} from 'react-native';
+import {RNCamera} from 'react-native-camera';
+
+class PhotoScreen extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      cameraOn: false,
+    };
+  }
+
+  takePicture = async () => {
+    if (this.camera) {
+      const options = {quality: 0.5, base64: true};
+      const data = await this.camera.takePictureAsync(options);
+      console.log(data.uri);
+    }
+  };
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <React.Fragment>
+          <RNCamera
```

Comment:
maybe good idea would be to move this settings to separated config? component is unreadable because of a lot number of config lines